### ISSUE TITLE
win-x64 publish with Core3 Preview and VS2019 results in ""CheckEmbeddedRootDescriptor" task was not given a value for the required parameter "AssemblyPath""

### ISSUE THREAD

**Issue by OliaG** [mono/linker#428]
_From @stefanov-stefan on December 5, 2018 16:11_

I have been trying to publish a plain dotnet new winforms app by following the steps [here]([url](https://github.com/dotnet/core/blob/master/samples/linker-instructions.md)), but I end up with error: error MSB4044: The "CheckEmbeddedRootDescriptor" task was not given a value for the required parameter "AssemblyPath".

See attached image for the whole process. 

I am running VS2019 Preview and have the following SDKs installed: https://www.screencast.com/t/29MbjIeY04 

Any advice on how I can overcome this?
![linker_issue](https://user-images.githubusercontent.com/1288402/49526561-be966b80-f8b8-11e8-9af1-ab64f18b0801.png)

_Copied from original issue: dotnet/winforms#210_

**Comment by jeffschwMSFT** [comment]
This issue is related to the SDK drifting slightly from our current state.  Our goal in an upcoming preview for .NET Core 3.0 is to integrate into the SDK to help eliminate these sorts of issues.

cc @sbomer @cshung

**Comment by wieslawsoltes** [comment]
Have same issue when using .NET Core 3.0 (3.0.100-preview-010184)
```
2019-02-01T18:15:04.4265533Z ##[error]C:\Users\VssAdministrator\.nuget\packages\illink.tasks\0.1.5-preview-1841731\build\ILLink.Tasks.targets(451,5): error MSB4044: The "CheckEmbeddedRootDescriptor" task was not given a value for the required parameter "AssemblyPath". [D:\a\1\s\src\ThemeEditor\ThemeEditor.csproj]
2019-02-01T18:15:04.4266070Z ##[error]Process 'dotnet.exe' exited with code 1.
```

**Comment by cup** [comment]
https://github.com/dotnet/core/issues/2208
https://github.com/dotnet/coreclr/issues/24397
https://github.com/dotnet/winforms/issues/210

https://github.com/mono/linker/issues/428

**Comment by marek-safar** [comment]
@sbomer was this issue already addressed?

**Comment by sbomer** [comment]
Yes, this should be fixed in the 3.0 SDK with `PublishTrimmed`.
